# HG changeset patch
# User Rik <rik@octave.org>
# Date 1478187564 25200
#      Thu Nov 03 08:39:24 2016 -0700
# Branch stable
# Node ID 9e415e57a6a7d4b529ceb4e823d3ab2957567162
# Parent  e07fa946f7631c89cda700bbaff9f3197b1ca247
doc: Clarify docstring for popen2 (bug #49524).

* syscalls.cc (Fpopen2): Document that second argument can be a
character matrix or cell array of strings.

diff --git a/libinterp/corefcn/syscalls.cc b/libinterp/corefcn/syscalls.cc
--- a/libinterp/corefcn/syscalls.cc
+++ b/libinterp/corefcn/syscalls.cc
@@ -201,17 +201,17 @@ error message.
 }
 
 DEFUNX ("popen2", Fpopen2, args, ,
         doc: /* -*- texinfo -*-
 @deftypefn {} {[@var{in}, @var{out}, @var{pid}] =} popen2 (@var{command}, @var{args})
 Start a subprocess with two-way communication.
 
 The name of the process is given by @var{command}, and @var{args} is an
-array of strings containing options for the command.
+array or cell array of strings containing options for the command.
 
 The file identifiers for the input and output streams of the subprocess are
 returned in @var{in} and @var{out}.  If execution of the command is
 successful, @var{pid} contains the process ID of the subprocess.  Otherwise,
 @var{pid} is @minus{}1.
 
 For example:
 
